test/metabase/pulse_test.clj

```diff
@@ -352,11 +352,13 @@
       :attachment-name "image.png",
       :channel-id "FOO",
       :fallback "Test card 2"}]}
+   true
    true]
   (slack-test-setup
    (let [[slack-data] (send-pulse! (retrieve-pulse pulse-id))]
      [(thunk->boolean slack-data)
-      (every? produces-bytes? (:attachments slack-data))])))
+      (< 10000 (-> slack-data :attachments first attachment-byte-count))
```

Comment:
@camsaul This is bad, but I don't have a better alternative. We take a small amount of HTML with one of our graphs inside it and render that to a PNG. That PNG gets Base64 encoded, put into a string and that's what we return. It either gets displayed on the Pulses page as a preview or included as a slack attachment.

The bug was just omitted the graph part of that (the thing that matters most). My test before made sure there were bytes coming back, but that's it, so it didn't catch the regression. The best I could come up with to help sniff it out next time is the size of the byte array. In the old version with a bug there were fewer bytes than if it was functioning correctly. The exact number is a little higher than these, but it's pretty close.

Any other ideas?